www/on/npm/%package.spt

```diff
@@ -56,7 +56,7 @@ if user.participant:
 {% if user.ANON %}
     <p class="instructions">
         {{ _( 'Claim the {package_link} npm package on Gratipay:'
-            , package_link=('<a href="' + package.remote_human_url + '">' + package_name + '</a>')|safe
+            , package_link=('<a href="' + package.remote_human_url + '">' + package.name + '</a>')|safe
```

Comment:
For the record, it's possible to inject links safely into translations, e.g.:

```
, package_link='<a href="%s">%s</a>'|safe % (package.remote_human_url, package.name)
```

That way the string passed to `|safe` is a constant, and the interpolated variables are automatically escaped (`|safe` is an alias for [markupsafe](https://github.com/pallets/markupsafe)'s `Markup()`).

Reply:
Nice, thanks! PR in #4585.